README.md

```diff
@@ -86,17 +95,20 @@ The enumerated backends are:
   implementations. We recommend starting with this if you're just starting with
   GUAC!
 
-- [arangoDB (supported, incomplete,
+- [ent (supported, complete
+  optimized)](https://github.com/guacsec/guac/tree/main/pkg/assembler/backends/ent)
+  with [PostgreSQL](https://www.postgresql.org/): a persistent backend based on
+  [Entity Framework for Go](https://entgo.io/) that can run on various SQL
+  backends. PostgreSQL is supported. Others, such as
```

Comment:
Might be good to be more explicit here. e.g. instead of "...various SQL backends. PostgreSQL is support. Others, such as..." we could say "...various SQL backends. GUAC only supports ent with PostgreSQL. Other ent backends such as..."

Reply:
Updated